templates/template-dominator/src/app.rs.lte

```diff
@@ -0,0 +1,124 @@
+                }))
+            }))
+            .child(html!("div", {
+                .children(&mut [
+                    html!("input" => HtmlInputElement, {
+                        .prop_signal("value", state.input.signal_cloned())
+    
+                        .with_node!(element => {
+                            .event(clone!(app => move |_: events::Input| {
+                                log::info!("input: {}", element.value());
+                                app.input.set(element.value());
+                            }))
+                        })
+                    }),
+    
+                    html!("button", {
+                        .text("Greet")
+                        .event(clone!(app => move |_: events::Click| {
+                            let app = app.clone();
+                            let input = app.input.lock_ref();
+    
+                            if *input == "" {
+                                app.clone().greet.set(None);
+    
+                            } else {
+
+                                let input = input.to_string();
+                                let args = Args {name: input.clone()};
+                                let app = app.clone();
+                                spawn_local(async move {
+                                    let name = invoke("greet", to_value(&args).unwrap()).await;
+                                    app.greet.set(name.as_string());
+                                });
+                            }
+                        }))
+                    }),
+    
+                    html!("div", {
+                        .text_signal(app.greet.signal_ref(|greet| format!("{}", greet.clone().unwrap_or_default())))
+                    }),
+                    
+                    html!("button", {
+                        .text("Greet2")
+                        .event(clone!(app => move |_: events::Click| {
+                            let app = app.clone();
+                            let input = app.input.lock_ref();
+    
+                            if *input == "" {
+                                app.clone().greet.set(None);
+    
+                            } else {
+
+                                let input = input.to_string();
+                                let args = Args {name: input.clone()};
+                                let app = app.clone();
+                                spawn_local(async move {
+                                    invoke2("greet2").await;
+                                    // app.greet.set(name.as_string());
+                                });
+                            }
+                        }))
+                    }),
+    
+                    html!("div", {
+                        .text_signal(app.greet.signal_ref(|greet| format!("{}", greet.clone().unwrap_or_default())))
+                    }),
+                    
+                ])
+            }))
+        })
+    }
+}
```

Comment:
We like to give a similar structure in all the templates so this should be changed to match other templates, the final result should look something like this 
![image](https://github.com/tauri-apps/create-tauri-app/assets/48618675/9f003e70-34af-4e83-8343-bc6f514e6e94)